app/views/miq_policy/_action_options.html.haml

```diff
@@ -68,12 +68,8 @@
       .form-group
         .col-md-8
           #action_tags_treebox.treeview-pf-hover.treeview-pf-select
```

Comment:
Either this whole line doesn't belong here anymore, or just the id, but `shared/tree` already creates a `*treebox`- in this case `cat_treebox`, and we don't want two :). (`action_tags_treebox` doesn't have any other references so can be safely removed)